[Bug 18366] Enable control-tab on Mac OS

bhall2001 opened the pull request:
Allow use of control-tab & control-shift-tab on Mac OS to cycle between tabs in the Script Editor.

bhall2001 committed:
[Bug 18366] Enable control-tab on Mac OS

Allow use of control-tab & control-shift-tab on Mac OS to cycle between tabs in the Script Editor.

livecodeali commented:
@bhall2001 Thanks for this! Is there any reason why it's restricted to MacOS?
There is a table in the docs for keyboard shortcuts relating to the script editor (https://github.com/livecode/livecode-ide/blob/develop/Documentation/guides/The%20LiveCode%20IDE.md#keyboard-shortcuts) so this should be added to that.

mwieder commented:
@livecodeali not necessarily restricted to MacOS: control-tab and shift-control-tab are already supported for cycling through script editor tabs on linux, so the patch isn't necessary there. I'm too lazy to test Windows.

livecodeali commented:
Ah makes sense, I had misread the patch slightly. In that case, it just needs adding to the shortcuts list in the guide.

livecodeali commented:
@bhall2001 As far as I'm concerned this is fine to merge once the shortcut has been added to the shortcuts table in the guides

bhall2001 commented:
I was surprised to see this shortcut was not already in the Docs. And to be clear, I have not changed any of the existing functionality for Linux and Windows. I replaced command-tab with control-tab on MacOS since command-tab is caught by the OS and never reaches LC.

bhall2001 committed:
Added Shortcut entry in Documentation

Added the shortcut documentation to the Editor section of the IDE documentation.

bhall2001 committed:
Corrected Linux/Windows to Command-tab

livecodeali commented:
@bhall2001 Thanks! I'm a little confused about that last commit - why is it not Control + tab on Windows/Linux?

bhall2001 committed:
Revert "Corrected Linux/Windows to Command-tab"

This reverts commit afd8a4a8f5bddf9c33770d1ec277f4fd1e54285e.

livecodeali commented:
@livecode-vulcan review ok 81cd8a8

livecode-vulcan commented:
💙 review by @livecodeali ok 81cd8a84df73c4cb57d1e5865078e636c3a61e71
